go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/kr/pretty v0.2.1
 	github.com/kylelemons/godebug v1.1.0
 	github.com/lestrrat-go/jwx v1.2.0
-	github.com/lucas-clemente/quic-go v0.21.0
+	github.com/lucas-clemente/quic-go v0.20.0
```

Comment:
Is there a reason to not move back to 0.20.1? A quick test show the later release still works.

Reply:
https://github.com/lucas-clemente/quic-go/issues/3194

Reply:
$ go version
go version go1.16.5 windows/amd64

Reply:
@hzxuzhonghu Note my comment was 0.20.1, not 0.21.0. I understand 0.21.0 has issues. I was asking why we shouldn't downgrade to the latest 0.20.x release.

Reply:
Oh, i missed v0.20.1

Reply:
done @ericvn  Thanks